DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzSupportToolsBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * Symfony extension configuration definition for ezplatform_support_tools extension.
+ *
+ * @internal
+ */
+final class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('ezplatform_support_tools');
+
+        $rootNode
+            ->children()
+                ->arrayNode('system_info')
+                    ->info('System info configuration, provided by "ez-support-tools" package')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('powered_by')
+                            ->info('Control if system should generate a powered by header to promote eZ Platform usage')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->booleanNode('enabled')
+                                    ->info('Activate/Deactivate powered by header')
+                                    ->defaultValue(true)
+                                ->end()
+                                ->enumNode('release')
+                                    ->info('Sets how much of version info is exposed in powered by. Examples: major: "2", minor: "2.5", ~: ""')
+                                    ->values(['major', 'minor', null])
```

Comment:
I think it should be presented in the order of amount of information that is presented, i.e. `minor` -> `major` -> none (`null`). Maybe also explicitly add `none`?

Reply:
`~` is null, but indeed it's not that obvious maybe, did you mean that we should change the line above to something like
```
>info('Sets how much of version info is exposed in powered by. Examples: major: "2", minor: "2.5", null: ""')
```

Reply:
Something like that. Although as I said, I'd order it by the amount of information exposed.

Reply:
ah, and you also mentioned `none`, ok so something like this then:
```
>info('Sets how much of version info is exposed in powered by. Examples:  minor: "2.5", major: "2", 'none': ""')
->values(['major', 'minor', 'none, 'null])
```

Reply:
> ah, and you also mentioned `none`, ok so something like this then:
> 
> ```
> >info('Sets how much of version info is exposed in powered by. Examples:  minor: "2.5", major: "2", none: ""')
> ->values(['major', 'minor', 'none', null])
> ```

@andrerom do we still need `null` if there's `none` option?

Reply:
No, I can drop it

Reply:
Fixed in ee27032